source/iface/device.rst

```diff
@@ -296,7 +296,8 @@ See `SYCL Specification <https://www.khronos.org/registry/SYCL/specs/sycl-2020/h
     all          // Maps to OpenCL CL_DEVICE_TYPE_ALL
   };
 
-See platform :ref:`platform-get_devices` and device :ref:`device-get_devices`.
+/*See platform :ref:`platform-get_devices` and device
+:ref:`device-get_devices`.*/
```

Comment:
Is this just a reminder for yourself to update this later?  If that is the case, it might be better to put the word TODO here, so it's obvious that it needs to be fixed later.

Note that this is not how comments work in RST, so this text is getting emitted in the HTML.  If you want to make it a comment, see this SO post: https://stackoverflow.com/questions/4783814/how-to-comment-a-string-in-restructured-text

Reply:
Fixed with : https://github.com/KhronosGroup/SYCL_Reference/commit/1dfdf2e32a01badcbd9c66be95c200dc1768d4a4